transmogrifier/sources/xml/marc.py

```diff
@@ -850,6 +650,104 @@ def _get_holdings_electronic_items(
                     note=holding_note,
                 )
 
+    @classmethod
+    def get_identifiers(cls, source_record: Tag) -> list[timdex.Identifier] | None:
+        identifiers = []
+        identifier_marc_fields = [
+            {
+                "tag": "010",
+                "subfields": "a",
+                "kind": "LCCN",
+            },
+            {
+                "tag": "020",
+                "subfields": "aq",
+                "kind": "ISBN",
+            },
+            {
+                "tag": "022",
+                "subfields": "a",
+                "kind": "ISSN",
+            },
+            {
+                "tag": "024",
+                "subfields": "aq2",
+                "kind": "Other Identifier",
+            },
+            {
+                "tag": "035",
+                "subfields": "a",
+                "kind": "OCLC Number",
+            },
+        ]
+        for identifier_marc_field in identifier_marc_fields:
+            identifiers.extend(
+                [
+                    timdex.Identifier(
+                        value=identifier.strip().replace("(OCoLC)", ""),
+                        kind=identifier_marc_field["kind"],
+                    )
+                    for datafield in source_record.find_all(
+                        "datafield", tag=identifier_marc_field["tag"]
+                    )
+                    if (
+                        identifier := (
+                            cls.create_subfield_value_string_from_datafield(
+                                datafield,
+                                identifier_marc_field["subfields"],
+                                ". ",
+                            )
+                        )
+                    )
+                ]
+            )
+        return identifiers or None
+
+    @classmethod
+    def get_languages(cls, source_record: Tag) -> list[str] | None:
+
+        languages = []
+        language_codes: list[str] = []
+
+        # get language codes from control field 008/35-37
+        if fixed_language_value := cls._get_control_field(source_record)[35:38].strip():
+            language_codes.append(fixed_language_value)
+
+        # get language codes from data field 041
+        for datafield in source_record.find_all("datafield", tag="041"):
+            language_codes.extend(
+                cls.create_subfield_value_list_from_datafield(datafield, "abdefghjkmn")
+            )
+
+        languages.extend(cls._get_language_names(source_record, language_codes))
+        languages.extend(cls._get_language_notes(source_record))
+        return languages or None
+
+    @classmethod
+    def _get_language_names(
+        cls, source_record: Tag, language_codes: list[str]
+    ) -> list[str]:
+        return [
+            language_name
+            for language_code in list(dict.fromkeys(language_codes))
+            if (
+                language_name := Marc.loc_crosswalk_code_to_name(
```

Comment:
I think for a `@classmethod`, we could use `cls.loc_crosswalk_code_to_name()` here?  I find it more idiomatic than calling the class by name.

Reply:
Totally agree with that

Reply:
Good catch! I meant to do this throughout the transform, but I guess I missed a few. Update made!